### ISSUE TITLE
Spaces overview tab navigation

### ISSUE THREAD

**Issue by fschade** [owncloud/web#9633]
if i navigate to a space with the keyboard, press tab again at the point where the preview is active, the focus ends up in nirvana, after another click the title is selected.

means: 3 tabs are necessary to get from the preview to the title, i hope you get the point, hard to show in the vid.

https://github.com/owncloud/web/assets/49308105/fef440bc-eafa-4fbb-94c4-cc92a6aa27ff

**Comment by tbsbdr** [comment]
I understood 2 issues, can you confirm I got it right?

1. Focus of checkbox is not visible ("nirvana")
2. Tab on the preview (Nr. 24 in the screenshot) is unnecessary and should be removed

![screenshot_001608](https://github.com/owncloud/web/assets/26610733/d53bcbf8-85aa-448c-912c-8cb6acd971a2)
(In firefox you can show the tab-order)

**Comment by fschade** [comment]
1.) yes it is, but hitting enter does not change the state, checked it via 'document.activeElement'
2.) yes 👍

**Comment by JammingBen** [comment]
Fixed via https://github.com/owncloud/web/pull/9670.
